bin/docker.sh

```diff
@@ -8,15 +8,17 @@ NETWORK_NAME="tus-php-network"
 
 # Build base image
 BASE_PATH="docker/base/"
+DOCKERFILENAME="${BASE_PATH}Dockerfile"
 
-docker build -t tus-php-base \
-    -f ${BASE_PATH}Dockerfile ${BASE_PATH}
+docker build \
+  -t tus-php-base \
+  -f ${DOCKERFILENAME} ${BASE_PATH}
 
 # Build client and server
 COMPOSE_FILE="docker/docker-compose.yml"
 
 docker-compose -p tus-php -f ${COMPOSE_FILE} down
 docker-compose -p tus-php -f ${COMPOSE_FILE} up --build --remove-orphans -d
 
-# Create uploads dir
 mkdir -p uploads
+chmod 777 uploads
```

Comment:
I think its better to restrict the permission to owner instead of setting it to 777

```bash
chown -R www-data:www-data uploads
```

Reply:
Yeah this seems more appropriate.

Reply:
I had to change it to `chown www-data:root -R uploads` as nginx and php process is started by root